rst/cartridge_dev.rst

```diff
@@ -930,101 +930,215 @@ exceptions relative to the rest of the templates does not matter, while in
+------------------------
 
-After you've developed your application locally, you can deploy
+After you've developed your Tarantool Cartridge application locally, you can deploy
 it to a test or production environment.
 
-"Deploy" includes packing the application into a specific distribution format,
-installing to the target system, and running the application.
+Deploying includes:
+
+*   packing the application into a specific distribution format
+*   installing it to the target server
+*   running the application.
 
 You have four options to deploy a Tarantool Cartridge application:
 
-* as an :ref:`rpm <cartridge-deploy-rpm>` package (for production);
-* as a :ref:`deb <cartridge-deploy-deb>` package (for production);
-* as a :ref:`tar+gz <cartridge-deploy-tgz>` archive (for testing,
-  or as a workaround for production if root access is unavailable).
-* :ref:`from sources <cartridge-deploy-rock>` (for local testing only).
+*   as an :ref:`RPM <cartridge-deploy-rpm>` package (for production)
+*   as a :ref:`DEB <cartridge-deploy-deb>` package (for production)
+*   as a :ref:`tar+gz <cartridge-deploy-tgz>` archive (for testing
+    or as a workaround for production if root access is unavailable)
+*   :ref:`from sources <cartridge-deploy-rock>` (for local testing only).
 
 .. _cartridge-deploy-rpm:
 .. _cartridge-deploy-deb:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Deploying as an rpm or deb package
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Deploying as an RPM or DEB package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The choice between DEB and RPM depends on the package manager of the target OS.
-For example, DEB is native for Debian Linux, and RPM -- for CentOS.
+DEB is used for Debian Linux and its derivatives, and RPM—for CentOS/RHEL and
+other RPM-based `Linux distributions <https://en.wikipedia.org/wiki/List_of_Linux_distributions>`_.
 
-#. Pack the application into a distributable:
+..  important::
 
-   .. code-block:: console
+    If you use the `Tarantool Community Edition <https://www.tarantool.io/en/developers/>`_
+    while packing the application, the package will have a dependency
+    on this version of Tarantool.
+
+    In this case, on a target server, add the Tarantool repository for
+    the version equal or later than the one used for packing the application.
+    This lets a package manager install the dependency correctly.
+    See details for your OS on the
+    `Download <https://www.tarantool.io/en/download/>`_ page.
+
+For a production environment, it is recommended to use the ``systemd`` subsystem
+for managing the application instances and accessing log entries.
+
+To deploy your Tarantool Cartridge application:
+
+#.  Pack the application into a deliverable:
 
-       $ cartridge pack rpm APP_NAME
+    .. code-block:: console
+
+       $ cartridge pack rpm [APP_PATH] [--use-docker]
        # -- OR --
```

Comment:
Sign here is used incorrectly. It should be

```suggestion
       $ # -- OR --
```

Otherwise, it denotes command `-- OR --` (invalid) executed with superuser rights.